src/schema/core.graphql

```diff
@@ -219,12 +219,19 @@ type Account {
   storage: Storage!
 }
 
+"""
+Input object to select a block by offset.
+"""
+input LogFilter {
+  topics: [[Bytes32]]
```

Comment:
What do 'topics' represent, and what are all of the possible topics?

Reply:
Good catch. I'm enhancing the docs in the schema.